addons/account/models/account_invoice.py

```diff
@@ -391,7 +399,7 @@ def _compute_portal_url(self):
         for order in self:
             order.portal_url = '/my/invoices/%s' % (order.id)
 
-    @api.depends('state', 'journal_id', 'date_invoice')
+    @api.depends('state', 'journal_id', 'date_invoice', 'type')
```

Comment:
ok. gotcha: that's because it's used in _get_seq_number_next_stuff()

Reply:
Now, we have <a href="https://drive.google.com/file/d/1f2pPhkz0kr_cjTYjztLs3VN0MLpGtldu/view">option</a> to choose from a vendor bill and a credit note on vendor bill/credit note form views while creating it, so this change will help loading correct sequence(in case Dedicated Credit Note Sequence is configured in purchase journal) for bill/credit note based on it's type.